packages/Tag/styles.ts

```diff
@@ -71,6 +78,10 @@ export const Tag = styled.div<StyledTagProps & WuiProps>(
         }
       }
     }
+
+    &:hover {
```

Comment:
we need to activate the hover style if is a link or got an onClick property ?

Reply:
and add a cursor pointer?

Reply:
We already got a pointer for `onRemove` action but rest of the time, we don't need it because it's not a link no ?

Reply:
It can be a link (for example list of tags to go on a page) or can have an onClick to, we do not want the hover effect if is not clickable

Reply:
Examples has been added in the `onClick` section 😀